lib/pure/includes/oserr.nim

```diff
@@ -83,13 +83,19 @@ proc osErrorMsg*(errorCode: OSErrorCode): string =
         if formatMessageW(0x00000100 or 0x00001000 or 0x00000200,
                         nil, errorCode.int32, 0, addr(msgbuf), 0, nil) != 0'i32:
           result = $msgbuf
```

Comment:
I doubt this produces a null character that is not part of the string!

Reply:
Anything appended to this string was not showing in the exception logs printed to the console. I was assuming it was a null character but now I realize it could also have been a newline/carriage return because those also prevented following text from being displayed. I'll do more investigation.

Reply:
Yeah I was wrong, it's actually just `\n\r` at the end of the string. Is there a reason why only the first line of multi-line text is printed during the exception logs?

Reply:
Ah, it appears it's the test framework that is truncating the multi-line strings.